### PR TITLE
Ocp cos ra 5(1) status update

### PR DESCRIPTION
**Pull request by thamill**
updated status and text for RA-5(1) - not applicable. organizational procedure controls are out of scope for both products.

### CODE REVIEW

File: coreos-4/policies/RA-Risk_Assessment/component.yaml [hunk 1]
```diff
@@ -149,12 +149,11 @@
...
   covered_by: []
-  implementation_status: planned
+  implementation_status: not applicable
   narrative:
     - text: |
-        A control response is planned. Progress can be tracked via:
-
-        https://issues.redhat.com/browse/CMP-353
+        Organizational policy and procedures to employ and update the organizational
+        scanning appliance is outside the scope of Red Hat CoreOS configuration.
```
Comment (shawndwells) [inline coreos-4/policies/RA-Risk_Assessment/component.yaml]:
```suggestion
        Organizational policies and procedures to employ and an organizational
        scanning tool are outside the scope of Red Hat CoreOS configuration.
```

Comment (shawndwells) [reply coreos-4/policies/RA-Risk_Assessment/component.yaml]: In the spirit of getting /something/ out the door, made a few minor tweaks. Ultimately a complete control response will document the native tools CoreOS delivers for vulnerability scanning.

File: openshift-container-platform-4/policies/RA-Risk_Assessment/component.yaml [hunk 2]
```diff
@@ -149,12 +149,22 @@
...
+        scanning appliance is outside the scope of OpenShift configuration.
+        The Quay container registry that Red Hat recommends for container storage can be
+        integrated with the Clair vulnerability scanning tool
+        Documentation for adding a Clair container to perform scans:
+        https://access.redhat.com/documentation/en-us/red_hat_quay/2.9/html/manage_red_hat_quay/clair-initial-setup
+
+        Clair vulnerability scan data sources and drivers documentation:
+        https://github.com/quay/clair/blob/master/Documentation/drivers-and-data-sources.md
+
+        Documentation progress can be tracked at:
```
Comment (shawndwells) [inline openshift-container-platform-4/policies/RA-Risk_Assessment/component.yaml]:
```suggestion
        Organizational policies and procedures to employ an organizational
        scanning tool are outside the scope of OpenShift configuration.

        To aid OpenShift system administrators, the Quay container registry
        that Red Hat recommends for container storage can be
        integrated with the Clair vulnerability scanning tool.
        Documentation for adding a Clair container to perform scans can
        be found at:

        https://access.redhat.com/documentation/en-us/red_hat_quay/2.9/html/manage_red_hat_quay/clair-initial-setup

        Clair vulnerability scan data sources and drivers documentation:
        https://github.com/quay/clair/blob/master/Documentation/drivers-and-data-sources.md
```

Comment (shawndwells) [reply openshift-container-platform-4/policies/RA-Risk_Assessment/component.yaml]: Minor copy editing.

Is there a non-github link for the clair docs? Something from docs.redhat.com or the customer portal?

File: openshift-container-platform-4/policies/RA-Risk_Assessment/component.yaml [hunk 3]
```diff
@@ -149,12 +149,22 @@
...
 
-        https://issues.redhat.com/browse/CMP-353
+        https://issues.redhat.com/CMP-353
```
Comment (shawndwells) [inline openshift-container-platform-4/policies/RA-Risk_Assessment/component.yaml]:
```suggestion
```

File: coreos-4/policies/RA-Risk_Assessment/component.yaml (hunk 1 above)
Comment (thamill) [reply coreos-4/policies/RA-Risk_Assessment/component.yaml]: I agree - but how are we defining native? Anything available through the configured BaseOS, AppStream and EPEL repos? That seems to broaden the scope of potential audit questions immensely. It seems that the only actually native scanner for the COS/OCP stack will be the security operator that is currently in-progress.

**Comment by thamill** [comment]
@shawndwells I have used appropriate docs.redhat.com where possible - if there is no appropriate documentation to cover the topic on the redhat.com docs, I then add the github /docs folder for content.

**Comment by redhatrises** [comment]
The comments in this PR haven't been addressed or resolved in a few months. This PR looks frozen. Closing. Please re-open this PR when the comments are addressed appropriately.

**Pull request closed without merging**